### DISCUSSION TITLE
`BeatmapOptionsButton` text being split per line causes issues in other languages

### DISCUSSION THREAD

**Discussion by Joppe27** [ppy/osu#22767] [Q&A] [answered]
Dutch translation:
![afbeelding](https://user-images.githubusercontent.com/56885706/222465992-70d55732-152b-487e-9227-0e7127d1a3e6.png)

- Currently: "Bewerken beatmap"
- Expected: "Beatmap bewerken"

Due to the way the buttons are implemented, translating this correctly is not possible:
https://github.com/ppy/osu/blob/29be26b1504d7056b8182156f505a47519acf16e/osu.Game/Screens/Select/PlaySongSelect.cs#L50

These ones will also become a problem when localised:
https://github.com/ppy/osu/blob/29be26b1504d7056b8182156f505a47519acf16e/osu.Game/Screens/Select/SongSelect.cs#L312-L315

From a language standpoint I'd highly suggest making the entire sentence one `LocalisableString` instead of splitting them up by line, but looking at the code it seems like that was explicitly not done for reasons (probably wrapping?):
https://github.com/ppy/osu/blob/29be26b1504d7056b8182156f505a47519acf16e/osu.Game/Screens/Select/Options/BeatmapOptionsButton.cs#L142-L155

A much simpler change would be making each line localisable instead of just the top one, but I think that would be confusing for translators without context.

**Comment by bdach** [comment]
These buttons are going to get a redesign anyway. I'm not sure it's worth the time to go fix this.

**Reply by Joppe27** [reply]
Waiting is fine, I wasn't expecting an immediate fix anyway. I mainly opened this for awareness because the issue would've likely stayed under the radar since it's language-specific.

**Reply by Joehuu** [reply, marked as answer]
https://github.com/ppy/osu/pull/24712
